pex/bin/pex.py

```diff
@@ -629,6 +629,18 @@ def configure_clp():
         "  This option can be used multiple times.",
     )
 
+    parser.add_option(
+        "-w",
+        "--wheel-dist",
+        dest="wheel_dists",
+        metavar="FILE",
+        default=[],
+        type=str,
+        action="append",
+        help="Add a wheel distribution (.whl) from a local path into the generated .pex file."
+        "  This option can be used multiple times.",
```

Comment:
Might be helpful to explain more about the expectation. Can this be a `.whl` _file_, or it should be an expanded folder?

Reply:
Relative path or absolute path?

Reply:
I think the fact that the metavar says it should be a file is enough to distinguish that it isn't an expanded folder, and the fact that the .whl suffix is suggested. I'll make a change to indicate it can be relative or absolute, and to canonicalize any relative paths.

Reply:
Actually the `PEXBuilder.add_dist_location` method that does the work here already deals with relative paths. I'll change the comment.